pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1021,6 +1021,19 @@ type PrometheusSpec struct {
 	// stream.
 	QueryLogFile string `json:"queryLogFile,omitempty"`
 
+	// File to which scrape failures are logged.
+	// Reloading the configuration will reopen the file.
+	//
+	// If the filename has an empty path, e.g. 'file.log', The Prometheus Pods
+	// will mount the file into an emptyDir volume at `/var/log/prometheus`.
+	// If a full path is provided, e.g. '/var/log/prometheus/file.log', you
+	// must mount a volume in the specified directory and it must be writable.
+	// It requires Prometheus >= v2.55.0.
+	//
+	// +kubebuilder:validation:MinLength=1
+	// +optional
+	ScrapeFailureLogFile *string `json:"scrapeFailureLogFile,omitempty"`
```

Comment:
Should we move to the `CommonPrometheusFields` struct?

Reply:
Thanks @simonpasquier Got it. 
Move it to common struct